clang/lib/DPCT/CubRule.cpp

```diff
@@ -0,0 +1,1354 @@
+//===--------------- CubRule.cpp ------------------------------------------===//
```

Comment:
how about CUBAPIMigration.cpp,  keep the style alined with other file already exist, like: BLASAPIMigration.cpp, FFTAPIMigration.cpp

Reply:
+1